src/s1a_portapotty.tw2

```diff
@@ -9,73 +9,51 @@ Wait. It's...
-
-There, at the end of the beam of light, you spot a [[small object|Jail Cell (Unlit) - examine lighter]] glinting slightly in the dimness.
-
-::Jail Cell (Unlit) - examine lighter <2910,522>
-You reach down and grasp the small object. Its contours feel familiar to your fingers.
-
-**You got the Lighter**
-
-::Jail Cell (Unlit) - intro <200,227>
-When I awoke, everything was darkness. (if: $sniffedAir is true)[
+::Jail Cell (Unlit) - hub <202,344>
+(if: $justWokeUp is true)[When I awoke, everything was darkness.(set: $justWokeUp to false)](else:)[Everything around me was darkness. (if: $sniffedAir is true)[
 
-Well...darkness and shit.]
+Well...darkness and shit.]]
 
 >> [[I blinked my eyes, still heavy from sleep.|Jail Cell (Unlit) - look - fail]]
 >> [[I felt around for something concrete.|Jail Cell (Unlit) - feel]]
 >> [[I sniffed the stagnant air.|Jail Cell (Unlit) - smell]]
 
+::Jail Cell (Unlit) - look - fail <30,500>
+My eyelids felt like paperweights, hanging heavily over eyes that were dry as sandpaper. I couldn’t see shit
+
+[[I tried rubbing my face. No good.|Jail Cell (Unlit) - hub]]
+
 ::Jail Cell (Unlit) - feel <201,504>
 My arms flopped clumsily at my sides. They felt like rubber.
 
-I slapped around a bit. Felt my legs in front of me—I was sitting on something hard and warm. Plastic?
+I slapped around a bit. Felt my legs in front of me — I was sitting on something hard and warm. Plastic?
 
 [I flailed around some more]<trigger1|.
 
-|section2)[Thwack—my arms hit something hard on both sides. Something cool and hard—the walls were close in.
-
-And sticky?
+|section2)[<em>Thwack</em> — my arms hit something hard on either side. Cool, hard — the walls were close in. And sticky?
 
 Panic seized me. A scream stuck in my throat, too dry to vocalize.
 
 It was at that moment that I started to really [lose my shit]<trigger2|.]
 
 |section3)[I stood up,
-lost my balance,
-fell backward,
-banged my head,
-and dropped my elbow into [something moist]<trigger3|.]
+(click: "stood up")[(transition: "dissolve") +(transition-time: 1s)[<span style="padding-left: 2em;">lost my balance,</span>]]
+(click: "lost my balance")[(transition: "dissolve") +(transition-time: 1s)[<span style="padding-left: 4em;">fell backward.</span>
 
-|section4)[Water—a basin full of it. I sniffed; didn't seem to be anything off about it.
+(click: "fell backward")[(transition: "dissolve") +(transition-time: 1s)[I banged my head and dropped my elbow into [something moist]<trigger3|.]]]]]
+
+|section4)[Water - a basin full of it. I sniffed — nothing off about it.
 
 Without hesitating, I [splashed my face and rubbed my eyes hard]<trigger4|.]
 
-|section5)[No good. Still couldn't see anything. Except...
+|section5)[No good. Still couldn't see anything. Except for...
 
 [>> [a faint beam of moonlight draped across the cell walls.]<trigger5a|]<prompt5a|
 [>> [a glint of metal on the floor.]<trigger5b|]<prompt5b|]
```

Comment:
Would love to see the "Lighter obtained" text appear on its own click rather than with a block of other text. Maybe a good place for a pulse transition?

Reply:
Cool, good suggestion! I added this in — let me know what you think!